### ISSUE TITLE
Package request: The Dark Mod

### ISSUE THREAD

**Issue by avoidr** [voidlinux/void-packages#13398]
Stealth Gaming in a Gothic Steampunk World
http://www.thedarkmod.com/

**Comment by avoidr** [comment]
Thank you, still!

**Comment by Johnnynator** [comment]
The Dark Mod got a new release with finally 64 bit support. I'll take a look if I can get it working now.

**Comment by Johnnynator** [comment]
Getting it running will take a bit longer since the build system will need some (a lot) patching, because it wants to link statically against some shipped libraries that are compiled without support for relocation (and other problems).

**Comment by avoidr** [comment]
Thank you for your effort.

Have you seen @abenson's work? Maybe it is useful to you.

**Comment by Johnnynator** [comment]
Yep I saw @abenson's work. But nothing that helps me with my current struggles.

**Comment by avoidr** [comment]
That was quick. With my machine I'd probably be still compiling 5 hours from now.

You're probably more experienced than me, anyway, but I suppose the data should be a separate package.

**Comment by Johnnynator** [comment]
At least the x86_64-musl binary is broken (segfaults when loading a mission).
Will also need additional work on getting it to use paths that are not all inside the same folder as the binary. 

If anyone would test if it works on either x86_64 or i686 would be helpful. 
Just clone my branch https://github.com/Johnnynator/void-packages/tree/thedarkmod and copy the binary into the data folder, you get when installing the game manually from their downloads page.

**Comment by abenson** [comment]
Segfault at same point on x86_64 (glibc).

**Comment by Johnnynator** [comment]
abenson tested it already and I for the information I wanted.
